### PR TITLE
Adding developer options to entry point: these are set in .env 

### PR DESCRIPTION
**Pull request by boekhorstb1**
Currently not sure how I can test this

### CODE REVIEW

**Comment by boekhorstb1** [comment]
OK, current code is working for the base installation.
I will test once more on Monday if it works for the Groupware, but I think it is fine.

Will change maintaina/deployments according and send a link when that is done (probably monday)

**Comment by boekhorstb1** [comment]
Updated [maintaina/deployments](https://github.com/maintaina/deployments/pull/9) accordingly

**Comment by boekhorstb1** [comment]
checked also for groupware-webmail and all is working, at least on my machine ;)

File: base/bin/entrypoint.sh [hunk 1]
```diff
@@ -93,20 +93,22 @@ if [[ -v HORDE_ADMIN_USER ]]; then
...
+	PHP_VERSION=$(php -r "echo PHP_VERSION;")
+	
+	if [[ $PHP_VERSION == 7.* ]]; then
+		zypper -n in php7-xdebug
+	fi
+	if [[ $PHP_VERSION == 8.* ]]; then
+		zypper -n in php8-xdebug
+	fi
+    zypper -n in vim wget mc iputils curl less bind-utils;
+	echo "ending installatin of developer tools"
```
Comment (ralflang) [inline base/bin/entrypoint.sh]: installatin -> installation

File: base/bin/entrypoint.sh [hunk 2]
```diff
@@ -93,20 +93,22 @@ if [[ -v HORDE_ADMIN_USER ]]; then
...
+	echo "developer mode is enabled: starting installation of developer tools"
+	PHP_VERSION=$(php -r "echo PHP_VERSION;")
+	
+	if [[ $PHP_VERSION == 7.* ]]; then
+		zypper -n in php7-xdebug
+	fi
+	if [[ $PHP_VERSION == 8.* ]]; then
+		zypper -n in php8-xdebug
+	fi
+    zypper -n in vim wget mc iputils curl less bind-utils;
```
Comment (ralflang) [inline base/bin/entrypoint.sh]: Add a variable to add more/custom software.

**Review by ralflang — CHANGES_REQUESTED** [review]

File: base/bin/entrypoint.sh (hunk 1 above)
Comment (boekhorstb1) [reply base/bin/entrypoint.sh]: ok ich hab die Rechtschreibung angepasst

File: base/bin/entrypoint.sh (hunk 2 above)
Comment (boekhorstb1) [reply base/bin/entrypoint.sh]: ok. Werde CUSTOM_TOOLS variable hinzufügen und testen

**Comment by boekhorstb1** [comment]
Ok, I added a CUSTOM_TOOLS variable. I also updated the .env.dist in the basic-setup here: https://github.com/maintaina/deployments/pull/9/files

**Review by ralflang — APPROVED** [review]
Looks good. Thank you.

**Pull request merged**